examples/smri_ants_registration.py

```diff
@@ -61,11 +62,12 @@
 
 """
 3. Define the parameters of the registration. Settings are
-saved in ``smri_ants_registration_settings.json``.
+found in the file ``smri_ants_registration_settings.json``
+distributed with the ``example_data`` of `nipype`.
 
 """
 
-reg = Registration(from_file='./smri_ants_registration_settings.json')
+reg = Registration(from_file=example_data('smri_ants_registration_settings.json'))
```

Comment:
moved this settings file from examples to the example data, since now I use it in tests